API/Controllers/PersonController.cs

```diff
@@ -236,17 +277,17 @@ public async Task<ActionResult<PersonDto>> MergePeople(PersonMergeDto dto)
     /// <summary>
     /// Ensure the alias is valid to be added. For example, the alias cannot be on another person or be the same as the current person name/alias.
     /// </summary>
-    /// <param name="personId"></param>
-    /// <param name="alias"></param>
+    /// <param name="dto">alias check request</param>
     /// <returns></returns>
-    [HttpGet("valid-alias")]
-    public async Task<ActionResult<bool>> IsValidAlias(int personId, string alias)
+    [HttpPost("valid-alias")]
+    public async Task<ActionResult<bool>> IsValidAlias(PersonAliasCheckDto dto)
     {
-        var person = await _unitOfWork.PersonRepository.GetPersonById(personId, PersonIncludes.Aliases);
+        var person = await _unitOfWork.PersonRepository.GetPersonById(dto.PersonId, PersonIncludes.Aliases);
         if (person == null) return NotFound();
 
-        var existingAlias = await _unitOfWork.PersonRepository.AnyAliasExist(alias);
-        return Ok(!existingAlias && person.NormalizedName != alias.ToNormalized());
+        var aliasIsName = dto.Name.ToNormalized() == dto.Alias.ToNormalized();
+        var existingAlias = await _unitOfWork.PersonRepository.AnyAliasExist(dto.Alias);
+        return Ok(!existingAlias && !aliasIsName);
```

Comment:
Try to get into the habit of adding a newline above return statements to make the code easier to navigate.